### DISCUSSION TITLE
How to make strikethrough text ?

### DISCUSSION THREAD

**Discussion by GaetanBaert** [fyne-io/fyne#4295] [Q&A] [answered]
Hello , 

I want to display strikethrough text in my app. 

I supposed I could get this with `widget.NewRichTextFromMarkdown("~~This is strikethrough text~~")`

But it gave \~\~This is strikethrough text\~\~ instead of ~~This is strikethrough text~~

Did I make something wrong, or maybe it is not implemented yet ?

**Comment by andydotxyz** [comment, marked as answer]
It is not implemented yet.

Also I don't think strikethrough is part of the base markdown spec?

**Reply by GaetanBaert** [reply]
It seems it is part of the Extended Syntax
https://www.markdownguide.org/extended-syntax/#strikethrough

I didn't find markdown spec unfortunately, it seems quite hard to find...

**Reply by andydotxyz** [reply]
Daring fireball published the original spec. I'm not sure if "Extended Markdown" has a definitive spec, GFM seems to be the most popular version.

Either way our markdown helper is just core spec at the moment

**Reply by GaetanBaert** [reply]
Should I make an "enhancement" issue to ask for the feature, or is it already planned ?

**Reply by andydotxyz** [reply]
Please do, if there is no issue here it's not planned!
